app/models/educator.rb

```diff
@@ -17,13 +17,14 @@ class Educator < ActiveRecord::Base
 
   validates :email, presence: true, uniqueness: true
 
-  validate :has_school_unless_districtwide,
-           :admin_gets_access_to_all_students,
-           :grade_level_access_is_array_of_strings,
-           :grade_level_strings_are_valid,
-           :grade_level_strings_are_uniq,
-           :grade_level_access_is_not_nil
```

Comment:
These look like they're all for validation (some read like authorization), so I moved them to private and prefixed them all with `validate`.

Reply:
Yes, these are all for validation. Question here: does prefixing with `validation` automatically run these as validations? Is this some ancient Rails magic you're invoking here? 🔮

Reply:
No I just renamed them :)